sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -365,14 +365,13 @@ case class Join(
 }
 
 /**
- * Append data to an existing table.
+ * Base trait for DataSourceV2 write commands
  */
-case class AppendData(
-    table: NamedRelation,
-    query: LogicalPlan,
-    isByName: Boolean) extends LogicalPlan {
+private[sql] trait V2WriteCommand extends Command {
```

Comment:
catalyst is already a private module. so it doesn't need `private[sql]`.

Reply:
I'm not sure that's true. Data types are defined in catalyst, but are part of the public API, right? Is it okay to leave this as-is to be safe?

Reply:
Okay, I can remove it. But I don't think it makes sense considering that this can't actually be a private module. Did I miss something there?

Reply:
Agreed. I'll stick to the convention for now.